pkg/kubelet/kubelet_network_test.go

```diff
@@ -136,6 +136,44 @@ func TestParseResolvConf(t *testing.T) {
 	}
 }
 
+func TestComposeDNSSearch(t *testing.T) {
+	testCases := []struct {
+		dnsNames     []string
+		hostNames    []string
+		resultSearch []string
+	}{
+		{[]string{"default.svc.cluster.local", "svc.cluster.local", "cluster.local"},
```

Comment:
the linewrapping makes these hard to visually compare.  Please put wrap at the open/close curlies for the struct